configure

```diff
@@ -39,16 +39,13 @@ sub_error()
     exit 1
 }
 
+# NOTE: Always quote your warning() messages
 warning()
 {
-    echo
-    echo "Warning: $*"
-}
-
-sub_warning()
-{
-    echo "  ... $*"
-    echo
+    >&2 echo
+    >&2 echo "Warning: $1"
+    [ -n "${2:-}" ] && >&2 echo " ... $2" # Built-in sub_warning()
```

Comment:
The "built-in sub_warning()" comment doesn't make sense to someone reading the code, because sub_warning doesn't exist. You could probably remove the comment altogether (once you've documented the parameters, see comment above).